### ISSUE TITLE
Cannot find type definition files - 2.0.0-rc.12

### ISSUE THREAD

**Issue by Kflash** [s-panferov/awesome-typescript-loader#177]
Hey! Nice work! I upgraded to v. 2.0.0-rc.12 and got some errors regarding type definition.

```
ERROR in [default]
Cannot find type definition file for 'chai'.
```

I guess this happen because `awesome-typescript-loader` are looking for the `typings` folder?

I'm running latest TypeScript 2.0 Pre (nigthly build). From TS 2.0 typings as we know it from before are all deprecated. Now type definition has to be installed through NPM `@types/chai`

You can read about it here: https://github.com/Microsoft/TypeScript/issues/9184

I guess this is not included in your code base yet?

Another issue I can see will happen is... what if the end-dev decide to exclude the `node_modules` folder?
With the new approach the new types are located here  `node-modules/@types`

**Comment by s-panferov** [comment]
@Kflash this case must be supported already. Could you please provide some more info about your setup?

**Comment by Kflash** [comment]
Not home atm but if you try my boilerplate with your latest version you
will see the errors when you bundle;

https://github.com/Kflash/sikky/blob/master/README.md

**Comment by s-panferov** [comment]
@Kflash thanks for the link, your case resolved in `2.0.0-rc.14`

**Comment by Kflash** [comment]
@s-panferov Nice job! :+1:
